### PR TITLE
Add section for scheduler support doc

### PR DESCRIPTION
**Pull request by dnwillia-work**
Added a new section in the user guide to cover scheduler and distributed parallel support.

### CODE REVIEW

**Review by ypatel-qa — APPROVED** [review]

File: doc/source/user_guide/launching_ansys_fluent.rst [hunk 1]
```diff
@@ -43,18 +44,110 @@ and set the floating point precision:
...
 .. code:: python
 
    solver_session = pyfluent.launch_fluent(precision='double', version='2d', mode="solver")
 
-Number of processors
-~~~~~~~~~~~~~~~~~~~~
-This example shows how you launch fluent in solution mode and also set the number of processors:
+Local parallel
+~~~~~~~~~~~~~~
+This example shows how you launch fluent in solution mode and also set the
```
Comment (seanpearsonuk) [inline doc/source/user_guide/launching_ansys_fluent.rst:56]:
```suggestion
This example shows you how to launch fluent in solution mode and also set the
```

File: doc/source/user_guide/launching_ansys_fluent.rst [hunk 2]
```diff
@@ -43,18 +44,110 @@ and set the floating point precision:
 
 Dimension
 ~~~~~~~~~
-This example shows how you launch Fluent in solution mode and also set the 
+This example shows how you launch Fluent in solution mode and also set the
```
Comment (seanpearsonuk) [inline doc/source/user_guide/launching_ansys_fluent.rst:47]:
```suggestion
This example shows you how to launch Fluent in solution mode and also set the
```

File: doc/source/user_guide/launching_ansys_fluent.rst [hunk 3]
```diff
@@ -43,18 +44,110 @@ and set the floating point precision:
...
+
+   solver_session = pyfluent.launch_fluent(precision='double', version='3d', processor_count=16,  mode="solver", additional_arguments="-cnf=m1:8,m2:8")
+
+
+Scheduler support
+-----------------
+When PyFluent is run within a job scheduler environment :func:`launch_fluent()
+<ansys.fluent.core.launcher.launcher.launch_fluent>` automatically determines
+the list of machines and core counts to start Fluent with. The supported
+scheduler environments include Univa Grid Engine (UGE), Load Sharing Facility
```
Comment (seanpearsonuk) [inline doc/source/user_guide/launching_ansys_fluent.rst:78]:
```suggestion
scheduler environments are Univa Grid Engine (UGE), Load Sharing Facility
```

File: doc/source/user_guide/launching_ansys_fluent.rst [hunk 4]
```diff
@@ -43,18 +44,110 @@ and set the floating point precision:
...
+   solver_session = pyfluent.launch_fluent(precision='double', version='3d', mode="solver")
+
+If you want to clamp the number of cores that Fluent is launched on you can
+provide the ``processor_count`` option:
+
+.. code:: python
+
+   solver_session = pyfluent.launch_fluent(precision='double', version='3d', processor_count=16, mode="solver")
+
+So, since the Slurm submit script example requested 32 total cores from the job
```
Comment (seanpearsonuk) [inline doc/source/user_guide/launching_ansys_fluent.rst:134]:
```suggestion
So, although the Slurm submit script example requested 32 total cores from the job
```

Comment (seanpearsonuk) [reply doc/source/user_guide/launching_ansys_fluent.rst:134]: ... or missing I'm misinterpreting, but I don't quite follow the logic as it's written.

File: doc/source/user_guide/launching_ansys_fluent.rst [hunk 5]
```diff
@@ -43,18 +44,110 @@ and set the floating point precision:
...
+scheduler this example would force Fluent execution on 16 of the cores. This is
```
Comment (seanpearsonuk) [inline doc/source/user_guide/launching_ansys_fluent.rst:135]:
```suggestion
scheduler, this example would force Fluent execution on 16 of the cores. This is
```

File: doc/source/user_guide/launching_ansys_fluent.rst (hunk 3 above)
Comment (dnwillia-work) [reply doc/source/user_guide/launching_ansys_fluent.rst:78]: There's more but I got them.

File: doc/source/user_guide/launching_ansys_fluent.rst (hunk 4 above)
Comment (dnwillia-work) [reply doc/source/user_guide/launching_ansys_fluent.rst:134]: Let me rework it.  It's a bit awkward the way it is.

**Review by seanpearsonuk — APPROVED** [review]

**Pull request merged**